cluster/examples/kubernetes/ceph/filesystem.yaml

```diff
@@ -26,6 +26,8 @@ spec:
         # gives a hint (%) to Ceph in terms of expected consumption of the total cluster capacity of a given pool
         # for more info: https://docs.ceph.com/docs/master/rados/operations/placement-groups/#specifying-expected-pool-size
         #targetSizeRatio: .5
+      #compressionMode: none | passive | aggressive | force
```

Comment:
How about uncommenting it and having the default be `none`? Then the setting is more visible and the comment can contain the options.

Reply:
Is there any drawback (performance?) to having the default compression mode being 'passive'? 
@liewegas  ?
(The paranoid in me would suspect a non-cooperating client might try to DoS us with in-compressible data, but that's a bit far fetched, no? there are other ways it can probably cause us to work hard).

Reply:
> How about uncommenting it and having the default be `none`? Then the setting is more visible and the comment can contain the options.

Done, I prefer them visible as well, otherwise it is lost in the comment noise at times.

As I changed these to be visible, I also changes the *-test.yaml files to add the required `none` values, such that these are tested during the integration tests.